packages/ndla-licenses/src/licenses.ts

```diff
@@ -413,7 +413,7 @@ export const ALL_ABBREVIATIONS = [
   "N/A",
 ] as const;
 
-export function getLicenseByAbbreviation(abbreviation: string, locale: Locale | string | undefined) {
+export function getLicenseByAbbreviation(abbreviation: string, locale?: Locale | string | undefined) {
```

Comment:
```suggestion
export function getLicenseByAbbreviation(abbreviation: string, locale?: Locale | string) {
```
Kunne fjernet undefined på slutten siden du bytter til _locale?_  Og i så fall burde kanskje gjøre samme med linje 393?

Reply:
Sender bare inn parametret istedenfor